hostwatch/hostwatch.go

```diff
@@ -19,6 +19,12 @@ const (
 
 	// DefaultMetadataAddress specifies the default value to use if nothing is specified
 	DefaultMetadataAddress = "169.254.169.250"
+	// ActiveState specifies the default value of active state of host in Ranceher Metadata
+	ActiveState = "activating"
```

Comment:
Need to add active to the list of active states, and inactive to the list of inactive state so reconcile logic considers them